main/multicall/index.ts

```diff
@@ -56,12 +56,17 @@ function buildCallData <R, T> (calls: Call<R, T>[]) {
   })
 }
 
-function getResultData (results: any, call: string[]) {
+function getResultData(results: any, call: string[]) {
   const [fnSignature] = call
   const callInterface = memoizedInterfaces[fnSignature]
   const fnName = getFunctionNameFromSignature(fnSignature)
-
-  return callInterface.decodeFunctionResult(fnName, results)
+  try {
+    return callInterface.decodeFunctionResult(fnName, results)
+  } catch (e) {
+    log.warn(`Failed to decode ${fnName} `, results)
```

Comment:
extra space here. also I find it makes it easier to read the logs if you label what you're printing. a cool trick to do that is use an object:

`log.warn(`Failed to decode ${fnName}`, { results })`